server/src/testFixtures/java/io/crate/testing/SQLExecutor.java

```diff
@@ -567,6 +564,28 @@ public Builder addTable(String createTableStmt, Settings settings) throws IOExce
             return this;
         }
 
+        public Builder closeTable(String tableName) throws IOException {
```

Comment:
couldn't find an easier way to keep state of 3 steps in unit tests - create/close/alter